### ISSUE TITLE
Script block missing

### ISSUE THREAD

**Issue by ToddismDotCom** [ubilabs/geocomplete#323]
It's the damdest thing:  i have a page with the following:

	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.0/jquery.min.js"></script>
<script src="https://maps.googleapis.com/maps/api/js?key=xxx&libraries=places"></script>
	<script src="jquery.geocomplete.js"></script>

When the page renders  	<script src="jquery.geocomplete.js"></script> is missing from the source  and  I get geocomplete is not a function.  

in the wild: 

http://www.platinumip.com/l2.htm

I'm at a loss.

**Comment by ToddismDotCom** [comment]
I hacked it by adding the jquery to the page.  No idea what's happening.
